.github/workflows/create-release.yml

```diff
@@ -31,6 +31,22 @@ jobs:
           python -m pip install bump2version
           echo -n "::set-output name=next_tag::"
           bump2version --list ${{ github.event.inputs.versionName }} | grep new_version | sed -r s,"^.*=",,
+      - name: Get main branch SHA
+        id: base-sha
+        run: echo "sha=$(git rev-parse origin/main)" >> $GITHUB_OUTPUT
+      - name: Get Python client file changes
+        id: changed-py
+        uses: tj-actions/changed-files@v44
```

Comment:
should we use `dorny/paths-filter` here (and also in `release-helper.yml`) to match the usage in the `covidcast-indicators` repo?  fewer dependencies and reinvented wheels make for easier support and maintenance.
https://github.com/cmu-delphi/covidcast-indicators/blob/a6ea00355d79a47cc09db8090f6e5121c718de0f/.github/workflows/create-release.yml#L32-L40
https://github.com/cmu-delphi/covidcast-indicators/blob/a6ea00355d79a47cc09db8090f6e5121c718de0f/.github/workflows/publish-release.yml#L31-L41

Reply:
I can give it a go, it should support the same features as `changed-files`.